### DISCUSSION TITLE
Problema de posicionamento no google, meu site tem erros?

### DISCUSSION THREAD

**Discussion by tardelipires** [frontendbr/forum#1474] [Perguntas e Respostas]
Olá pessoal, sou novo aqui. Estou com uma duvida e gostaria de saber se voces poderiam me ajudar. Tenho um blog no blogger há 10 anos. Antes meu blog aparecia com facilidade no google, hoje em dia, quase não aparece. Tenho 26 mil postagens, já que é um blog de noticias. as postagens estão todas indexadas. Recentemente tive que trocar o template para um mais novo e moderno. Alguem me disse que os codigos do meu blog poderia está com problema. Será que vocês poderiam dá uma olhada no codigo fonte para olhar se vocês percebem alguma coisa de errada com o mesmo? Sou leigo e entendo o básico de HTML.
meu site é o: www.catingueiraonline.com

**Comment by kvnol** [comment]
@tardelipires, beleza? Cara, aqui no fórum não costumamos a falar sobre SEO ou fazer consultoria, falamos mais sobre as tecnologias em si. Enfim.

Mas vou dar uma dica essencial, seu site está muito lento, tem muito conteúdo carregando de uma vez e está com uma nota péssima de performance. Se liga: https://developers.google.com/speed/pagespeed/insights/?hl=pt-br&url=https%3A%2F%2Fwww.catingueiraonline.com%2F&tab=mobile

É extremamente importante resolver isso.

Se quiser uma ajuda melhor de como resolver isso, entre em contato comigo por e-mail: me@kevinoliveira.com.br

abraço

**Comment by adamfoerster** [comment]
Um outro endereço útil que mostra um diagnóstico da página é o [Web.dev](https://web.dev)
Olha como ficou o resultado do seu site:
https://lighthouse-dot-webdotdevsite.appspot.com/lh/html?url=https://www.catingueiraonline.com/

**Comment by felipefialho** [comment]
@tardelipires Dá uma olhada aqui #88

**Comment by onlurking** [comment]
Eu sempre analiso SEO usando o [Seorch](https://seorch.eu/seo-check/www-catingueiraonline-com-2019-07-04-04-94-43.html), essa aqui é a [análise de uma de suas páginas](https://seorch.eu/seo-check/www-catingueiraonline-com-2019-07-04-04-77-78.html) por exemplo, eu também costumo implementar uma coisa chamada [JSON-LD](https://jsonld.com/news-article/) também nas que precisam de rankeamento, mas isso é algo relativamente avançado.

O que mais impacta SEO na realidade são duas coisas: se o site é responsivo e abre certo em um celular(o Google hoje em dia apenas olha pro conteúdo em mobile, ele não leva em conta a versão desktop para indexar) e Backlinks, que é quando outros sites bem rankeados no Google usam o seu site como referência(esse aqui em particular é o fator mais importante de todos e não tem haver com código).

Eu recomendo você buscar um especialista de SEO.

**Comment by tardelipires** [comment]
Fica agradecido pela atenção de todos. O problema é que tenho poucas condições, e sou leigo no assunto. Moro em uma cidade pequena do interior da Paraíba, o blog é regional e recebe um número razoável de acessos mas gera pouquinha renda, por isto que nunca sai do sistema blogger por que ficaria inviável para mim mandar construir um site e ainda ficar pagando hospedagem
